models/sklearn_extra_trees.py

```diff
@@ -0,0 +1,116 @@
+import datatable as dt
+import numpy as np
+from h2oaicore.models import CustomModel
+from sklearn.ensemble import ExtraTreesClassifier, ExtraTreesRegressor
+from sklearn.preprocessing import LabelEncoder, StandardScaler
+
+
+class ExtraTreesModel(CustomModel):
+
+    _regression = True
+    _binary = True
+    _multiclass = True
+
+    _display_name = "ExtraTrees"
+    _description = "Extra Trees Model based on sklearn"
+
+    # def make_instance(self):
+    #     return self.__class__._class(seed=self.random_state)
+
+    def set_default_params(self, accuracy=None, time_tolerance=None,
+                           interpretability=None, **kwargs):
+        n_jobs = -1
+        n_estimators = min(kwargs.get('n_estimators', 10), 1000)
```

Comment:
Change to match 0.22 n_estimators new default
```suggestion
        n_estimators = min(kwargs.get('n_estimators', 100), 1000)
```